src/main/kotlin/uk/gov/gdx/datashare/service/DataShareTopicService.kt

```diff
@@ -24,11 +24,12 @@ class DataShareTopicService(awsQueueService: AwsQueueService, private val object
   }
   private val domainEventsTopicClient by lazy { domainEventsTopic.snsClient }
 
-  fun sendGovEvent(eventId: UUID, occurredAt: LocalDateTime, eventType: String) {
+  fun sendGovEvent(eventId: UUID, consumer: String, occurredAt: LocalDateTime, eventType: String) {
```

Comment:
Adding consumer as possible idea for filtering topic by consumer, (used in the outbound example)